indico/htdocs/sass/modules/_papers.scss

```diff
@@ -122,6 +122,16 @@
         color: $light-black;
         padding: 20px 0 10px;
     }
+
+    .template-list {
+        font-size: 13pt;
```

Comment:
We use px/em everywhere else (except for legacy). I don't think we want to start using pt in a few places. `em` is probably the best choice here.